api/app/controllers/webhook.controller.ts

```diff
@@ -38,7 +42,7 @@ async function createCreditTransaction(review, amount) {
     await Transaction.create(creditTransaction, {
       transaction: dbTransaction,
     });
-    await userWallet.update(
+    await userWallet?.update(
```

Comment:
We should check if `userWallet` is defined to avoid optional chaining for db methods.